### PR TITLE
Fix #55: FontAwesome dynamically display icons.

### LINKED ISSUE #55: FontAwesome 5 Dynamically Display Page
Update the showcase page for FontAwesome 5.

https://github.com/primefaces/primefaces/issues/3056

I will submit a PR.

### PR DESCRIPTION
**Pull request by melloware**
Fix #55: FontAwesome dynamically display icons.

- Dynamically display icons
- Determine "fas" or "fab" by brand icons
- Dynamically show count of icons.

### CODE REVIEW

**Comment by jxmai** [comment]
Awesome work ! and the end result looks pretty good. :+1:

**Comment by jxmai** [comment]
Just one thing, the refresh and remove icons are missing. I know they are not part of the changes involved in this commit but they are in the same page.

**Comment by melloware** [comment]
Thanks for pointing that out.  All changes made and committed.

File: src/main/webapp/ui/data/datatable/paginator.xhtml [hunk 1]
```diff
@@ -75,7 +75,7 @@
         &lt;/p:column&gt;
         
         &lt;f:facet name="paginatorTopLeft"&gt;
-            &lt;p:commandButton type="button" icon="fa fa-refresh" /&gt;
+            &lt;p:commandButton type="button" icon="fa fa-times" /&gt;
```
Comment (jxmai) [inline src/main/webapp/ui/data/datatable/paginator.xhtml]: Should refresh migrate to `sync` instead of `times`  ?

**Comment by melloware** [comment]
You are right.  Fixed.

**Comment by ravihariharan** [comment]
@tandraschko @melloware  the version need to be updated (Font Awesome Version 5.0.2 #3107)
`public static final String VERSION = "v5.0.2";`

**Comment by ravihariharan** [comment]
@tandraschko @melloware  we are missing the 115 Awesome Icons Regular icons.

image

**Comment by melloware** [comment]
I will submit an updated PR.

**Pull request merged**